### PR TITLE
add windows switch option into parameter

### PR DESCRIPTION
**Pull request by internetztube**
```
$ npm run update platform=windows
```

### CODE REVIEW

File: lib/downloader.js [hunk 1]
```diff
@@ -73,4 +73,17 @@ function downloader(binDir, callback) {
...
-module.exports = downloader;
\ No newline at end of file
+function isWin() {
+    for (let i = 0; i < process.argv.length; i++) {
+        argument = process.argv[i].split('=');
+        if (argument.length == 2 && argument[0] === 'platform' && argument[1] === 'windows') {
+            return true;
+        }
+    }
+    if (process.platform === 'win32' || process.env.NODE_PLATFORM === 'windows') {
```
Comment (Kikobeats) [inline lib/downloader.js]: Can you move this condition over the top? I think it's more important the loop. Use the loop as fallback

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
         "video",
         "download"
     ],
-    "version": "1.12.2",
+    "version": "1.13.0",
```
Comment (Kikobeats) [inline package.json]: Please don't increment the version into the PR

**Comment by Kikobeats** [comment]
Addressed at https://github.com/przemyslawpluta/node-youtube-dl/pull/243, it will be available in the next major version, thanks for all!

**Comment by internetztube** [comment]
Thanks!

**Pull request closed without merging**